draft-ietf-quic-transport.md

```diff
@@ -2618,6 +2618,11 @@ on new STREAM frames.  Retransmission of data that has already been sent on
 STREAM frames is permitted.  An endpoint MAY also send MAX_STREAM_DATA and
 STOP_SENDING in this state.
 
+An application with a stream in this state may no longer care about data sent on
+the stream and may choose to reset the stream. To permit this behavior, a
```

Comment:
s/To permit this behavior, a/A/

Or just replace the first two sentences with "An application can decide to abandon a stream in this state.  An endpoint can send RST_STREAM for a stream that was closed with the FIN flag."  Keep the final offset piece.